src/utils/convertjson.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python2
+# encoding: utf-8
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import json
+import argparse
+import logging
+import sys
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('in_json', type=str,
```

Comment:
Can you modify `convertjson.py` to accept standard input? Then, we can avoid to have a tmp file.
We can even include this conversion step in `data2json.sh`, but it depends on how we prepare the multichannel inputs.